build.gradle

```diff
@@ -26,7 +26,7 @@ plugins {
   id 'io.spring.dependency-management' version '1.1.3'
   id 'org.springframework.boot' version '2.7.16'
   id 'com.github.ben-manes.versions' version '0.49.0'
-  id 'hmcts.ccd.sdk'  version '5.4.5'
+  id 'hmcts.ccd.sdk'  version '5.4.7'
```

Comment:
@sjamihmcts - it will be good to add a test case (integration, functional) to cover the date format issue.

Reply:
As discussed with Chris, we would add functional/integration tests separately as a different jira.